observations/atmosphere/aircraft.yaml.j2

```diff
@@ -7,146 +7,552 @@
+        obsfile: "{{atmosphere_obsdatain_path}}/{{atmosphere_obsdatain_prefix}}prepbufr.acft_profiles"
       obsgrouping:
-        group variables: ["stationIdentification"]
-        sort variable: "pressure"
-        sort order: "descending"
+        group variables: [stationIdentification]
+        sort variable: pressure
+        sort order: descending
     obsdataout:
       engine:
         type: H5File
-        obsfile: "{{atmosphere_obsdataout_path}}/{{atmosphere_obsdataout_prefix}}{{observation_from_jcb}}{{atmosphere_obsdataout_suffix}}"
+        #obsfile: "/work2/noaa/da/nesposito/BufrConverterTest_aircraft/diags/2021080100/diags_aircft_prepbufr.2021080100.nc" #NICKE test
+        obsfile: "{{atmosphere_obsdataout_path}}/{{atmosphere_obsdataout_prefix}}prepbufr.acft_profiles_{{atmosphere_obsdataout_suffix}}"
     io pool:
       max pool size: 1
-    simulated variables: [windEastward, windNorthward, airTemperature, specificHumidity]
-  #
+    simulated variables: [airTemperature, windEastward, windNorthward, specificHumidity]
 
   # Observation Operator
   # --------------------
   obs operator:
-    name: VertInterp
-  #
+    name: Composite
+    components:
+    - name: VertInterp
+      variables:
+      - name: windEastward
+      - name: windNorthward
+      # Hofx scaling
+      hofx scaling field: SurfaceWindScalingPressure
+      hofx scaling field group: DerivedVariables
 
-  # Observation Filters (QC)
-  # ------------------------
-  obs filters:
-    # Observation Range Sanity Check
-  - filter: Bounds Check
+    - name: VertInterp
+      observation alias file: "/work2/noaa/da/acollard/git/GDASApp/parm/atm/obs/testing/swell/obsop_name_map.yaml"
+      variables: [airTemperature ]
+
+    - name: VertInterp
+      variables:
+      - name: specificHumidity 
+
+  # Linear Observation Operator
+  # ---------------------------
+  linear obs operator:
+    name: Composite
+    components:
+    - name: VertInterp
+      variables:
+      - name: windEastward
+      - name: windNorthward
+
+      # Hofx scaling
+      hofx scaling field: SurfaceWindScalingPressure
+      hofx scaling field group: DerivedVariables
+
+    - name: VertInterp
+      observation alias file: "/work2/noaa/da/acollard/git/GDASApp/parm/atm/obs/testing/swell/obsop_name_map.yaml"
+      variables: [airTemperature] 
+
+    - name: VertInterp
+      variables:
+      - name: specificHumidity
+
+  # Observation Bias Correction (VarBC)
+  # -----------------------------------
+  obs bias:
+    input file: "{{atmosphere_obsbiasin_path}}/{{atmosphere_obsbiascovin_acft_suffix}}"
+    output file: "{{atmosphere_obsbiasin_path}}/{{atmosphere_obsbiascovout_acft_suffix}}"
+    input file: "/work2/noaa/da/nesposito/GDASApp_20250210/build/iodaconv/test/testrun/varbc/acft_out.nc4"  # NICKE test
+    output file: "/work2/noaa/da/esposito/ufoeval/GSIobserver_NE_ADC/2021080100/ioda/bc/acft_biasout.nc4"   # NICKE test
```

Comment:
Remove these test files

Reply:
We will  need to add the output file like the following the implementation below:
https://github.com/JCSDA-internal/ufo/blob/d5de93739f22fc9124cfcfde36831cc3ed80be7e/test/testinput/unit_tests/predictors/bias_coeff_cov_byrecord.yaml#L29-L40

We do not need to add the output file in this PR, but will need to add and test it in phase 2.